app/src/main/java/com/example/films/MovieDetailsActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.films
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class MovieDetailsActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_movie_details)
+
+        try {
+            this.supportActionBar!!.hide()
```

Comment:
maybe just this.supportActionBar?.hide()

Reply:
done